### PR TITLE
Added modMute Event

### PR DESCRIPTION
**Pull request by cosmetify**
_fingers crossed_

### CODE REVIEW

File: src/plugins/mutes.js [hunk 1]
```diff
@@ -18,5 +18,18 @@ export default function mutesPlugin () {
       mute,
       unmute
     })
+
+    function onModMute (ref) {
+      mp.emit('modMute', {
+        moderator: ref.m,
```
Comment (goto-bus-stop) [inline src/plugins/mutes.js]: I think this isn't right, I am seeing a `mi` property for the moderator ID in the browser:

Comment (cosmetify) [inline src/plugins/mutes.js]: Hmm, you're right. I was going off of this: https://github.com/plugcommunity/documentation/blob/master/api/events/backend_events/modMute.md

File: src/plugins/mutes.js [hunk 2]
```diff
@@ -18,5 +18,18 @@ export default function mutesPlugin () {
       mute,
       unmute
     })
+
+    function onModMute (ref) {
+      mp.emit('modMute', {
+        moderator: mp.user(ref.mi) || mp.wrapUser({ id: ref.mi, username: m }),
```
Comment (goto-bus-stop) [inline src/plugins/mutes.js]: `username: m` should be `username: ref.m` (and same with `t` on the next line)

Comment (cosmetify) [inline src/plugins/mutes.js]: Noticed that right before you commented :x

**Comment by goto-bus-stop** [comment]
Cool! Looking good now. Thanks again for picking these up :D

if you wanna update the plugcommunity docs too that'd be :100:

**Comment by JTBrinkmann** [comment]
If you are okay with me posting your userID there, then I can update the plugcommunity docs

**Comment by goto-bus-stop** [comment]
Or you could make one up 🤷‍♀️ I don't mind tho

**Pull request merged**